tests/bundle-test.ts

```diff
@@ -7,6 +7,9 @@
 
 /// <reference lib="dom" />
 
+import type { KeypairType } from '@polkadot/util-crypto/types'
+import { WsProvider, ApiPromise } from '@polkadot/api'
```

Comment:
I think this is the reason for the test to fail. We access everything kilt from window.kilt. I don’t know actually how we can access polkadot stuff in this case. Should we include it in the bundle somehow?

Reply:
it's one of the reasons. you also can't export apparently. I think we still need a connection builder then to make the bundle usable. or we re-export ApiPromise from the sdk package